test/runtests.jl

```diff
@@ -91,7 +93,9 @@ try
     NEW = GZip.GZLIB_VERSION > "1.2.3.9"
     pos = position(gzfile)
     NEW && (pos2 = position(gzfile,true))
-    @test_throws ErrorException seek(gzfile, 100)   # can't seek backwards on write
+    try 
+        @test_throws ErrorException seek(gzfile, 100)   # can't seek backwards on write
```

Comment:
so it didn't error at all, but it was supposed to. is this a bug or platform IO difference? I'm not sure.